crdppf/static/js/Crdppf/legalDocuments.js

```diff
@@ -27,27 +42,31 @@ Crdppf.docfilters = function(filter) {
         }
     }
     
-    if ('municipalitynb' in filter) {
-        if (filter['municipalitynb'] > 0) {
-            Crdppf.filterlist['municipalitynb'] = filter['municipalitynb'];
+    if ('cadastrenb' in filter) {
+        if (filter['cadastrenb'] > 0) {
+            Crdppf.filterlist['cadastrenb'] = filter['cadastrenb'];
         } else {
-            Crdppf.filterlist['municipalitynb'] = 0;
+            Crdppf.filterlist['cadastrenb'] = 0;
         }
     }
 
     Crdppf.legalDocuments.store.clearFilter();
     Crdppf.legalDocuments.store.filterBy(function (record) {
-        if (Crdppf.filterlist['municipalitynb'] > 0){
-            if (record.get('numcad') == Crdppf.filterlist['municipalitynb'] || record.get('numcad') == 0) {
-                for (var i = 0; i < Crdppf.filterlist['topic'].length; i++){
-                // if the topicid is in the filterlist show the corresponding documents
-                 if (record.get('topicfk') == Crdppf.filterlist['topic'][i].toString()) return record;
+        if (Crdppf.filterlist['cadastrenb'] > 0){
+            if (record.get('cadastrenb') == Crdppf.filterlist['cadastrenb'] || record.get('cadastrenb') == 0) {
+                if (Crdppf.filterlist['topic'].length > 0) {
+                    for (var i = 0; i < Crdppf.filterlist['topic'].length; i++){
+                    // if the topicid is in the filterlist show the corresponding documents
+                        if (record.get('origins').indexOf(Crdppf.filterlist['topic'][i].toString()) > -1) return record;
+                    }
+                } else {
+                    return record;
                 }
             }
         } else {
             for (var i = 0; i < Crdppf.filterlist['topic'].length; i++){
             // if the topicid is in the filterlist show the corresponding documents
-             if (record.get('topicfk') == Crdppf.filterlist['topic'][i].toString()) return record;
+                if (record.get('origins').indexOf(Crdppf.filterlist['topic'][i].toString()) > -1)  return record;
```

Comment:
I don't really like one line if conditions... But that might be a personal choice...